app/config/config.py

```diff
@@ -0,0 +1,10 @@
+from pymongo import MongoClient
+from app.config.settings import MONGODB_URI,CHANNEL_DB,DB_NAME,PORT 
+
+#creating a connection with MongoClient and its attributes from settings.py
+CLIENT = MongoClient(MONGODB_URI,PORT)
+#assigning database to a variable db  
+DB = CLIENT[DB_NAME]
+
+def channel_DB():
+    return DB[CHANNEL_DB]
```

Comment:
Why are you doing it like this? write it as a class. Plus, why is database related code in `config` package of all the things?

Reply:
This still not changed.